cassdegrees/templates/list.html

```diff
@@ -58,29 +60,28 @@
                                     {% if key != "id" %}
                                         <td>{{ item }}</td>
                                     {% else %}
-                                        <td><input title="Select" type="radio" name="id" value="{{ item }}"  /></td>
-                                        <td>(<a href="/api/model/{{ title.lower }}/{{ item }}/">View</a>)</td>
+                                        <td><input title="Select" type="checkbox" name="id" value="{{ item }}"  /></td>
+                                        <td>
+                                            (<a href="/api/model/{{ title.lower }}/{{ item }}/">View</a>)
+                                            <input class="btn-uni-grad" name="id" type ="submit" formaction="/{% if title == 'Degree' %}manage_programs{% elif title == 'Subplan' %}manage_subplans{% elif title == 'Course' %}manage_courses{% endif %}/?action=Edit" value="Edit {{ title }} (ID: {{ item }})">
```

Comment:
```suggestion
                                            <input class="btn-uni-grad" name="id" type ="submit" formaction="/manage_{% if title == 'Degree' %}programs{% elif title == 'Subplan' %}subplans{% elif title == 'Course' %}courses{% endif %}/?action=Edit" value="Edit">
```

Code minimisation and styling changes

Reply:
This is fine, but I kept ID there temporary so that the edit function worked. 
I need @Daniel-Jang to fix it since I couldn't find a way to parse ID over once the button was pressed. (ID is currently parsed by the checkbox being ticked - so if the checkbox isn't ticked the submit breaks because it gives the value of "edit" instead of "id"). Another problem is multi delete relies on the checkboxes id's (being ticked) so if I made another id function the delete function would break. 

I hope this explains what I'm trying to say, but I'll have a talk with Daniel about this.